functions/common/methods.ts

```
@@ -410,37 +467,1585 @@ export const getThisGamesAmbassadors = async (client: BikeTagClient, adminBikeTa
+}
+
+/** Validates moving a queue found image onto a specific past main tag round. */
+export const evaluateOrphanedQueueFoundForTarget = (
+  image: QueueStorageImage,
+  targetRound: number,
+  main: MainFolderContext,
+  simulatedQueue: Tag[] = [],
+): OrphanedQueueFoundCheck => {
+  const currentTag = main.currentTag
+  if (image.type !== 'found') {
+    return { structural: false, playerVerified: false, playerConflict: false, reasons: ['not a found image'] }
```

ESLint

[error] Replace `·structural:·false,·playerVerified:·false,·playerConflict:·false,·reasons:·['not·a·found·image']` with `⏎······structural:·false,⏎······playerVerified:·false,⏎······playerConflict:·false,⏎······reasons:·['not·a·found·image'],⏎···`

(prettier/prettier)